### PR TITLE
OADP-662: Fail VSB/VSR if backup/restore fails

### PR DESCRIPTION
**Pull request by eemcmullan**
image: `quay.io/emcmulla/data-mover:latest`

### CODE REVIEW

File: Dockerfile.ubi [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Build the manager binary
```
Comment (kaovilai) [inline Dockerfile.ubi:1]: Is this file part of OADP-662?

Comment (eemcmullan) [reply Dockerfile.ubi:1]: it is not

Comment (kaovilai) [reply Dockerfile.ubi:1]: So why include it here?

Comment (kaovilai) [reply Dockerfile.ubi:1]: >new Velero deps

Sounds more like a change to an existing dockerfile?

Including a new ubi image seems a scope for a separate PR 🤷

Comment (kaovilai) [reply Dockerfile.ubi:1]: At least the ubi dockerfile would be in addition to, not instead of, changing current dockerfile to work.

Comment (shubham-pampattiwar) [reply Dockerfile.ubi:1]: This PR needed Velero deps to be added to go.mod and the build was failing, we needed to update the builder, at the same time we wanted add the ubi dockerfile. Hence the change. Maybe @eemcmullan fix the existing non-ubi dockerfile as well along with the ubi dockerfile addition ?

Comment (eemcmullan) [reply Dockerfile.ubi:1]: @shubham-pampattiwar I can do that. I would like to keep the dockerfile changes in this PR though as they are needed to build these changes, as well as avoid rebasing.

Comment (shubham-pampattiwar) [reply Dockerfile.ubi:1]: +1, I am good with that.

Comment (kaovilai) [reply Dockerfile.ubi:1]: +1

**Review by kaovilai — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**